### PR TITLE
GT-939 Enable onboarding for French, Russian, and Indonesian

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

**Review by TheNoodleMoose — APPROVED** [review]
I assume the majority of the changes here are created automatically once you add the support for the locale in ```PageSet.kt``` right? Is it pulling from OneSky and then being generated by something else? If not I'd imagine it this would be a long process haha.

File: ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/PageSet.kt [hunk 1]
```diff
@@ -27,9 +27,12 @@ enum class PageSet(
         ),
         supportedLocales = setOf(
             Locale.ENGLISH,
-            LocaleCompat.forLanguageTag("zh-Hans"),
+            Locale("es"),
+            Locale.FRENCH,
+            Locale("in"),
+            Locale("ru"),
```
[the remark this answers is not in the document]
Comment (gyasistory) [reply ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/PageSet.kt:33]: @TheNoodleMoose French is a static variable that is stored in the Locale Class when the language you need is not present as a static variable you have to use the constructor to get your Locale.  Below is the link to the documentation for Local.

https://developer.android.com/reference/java/util/Locale

[the remark this answers is not in the document]
Comment (frett) [reply ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/PageSet.kt:33]: yup, I opted to use the Locale constants when available, it's such an odd set of which constants exist and which don't that date back to the early versions of Java.

[the remark this answers is not in the document]
Comment (TheNoodleMoose) [reply ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/PageSet.kt:33]: That was probably going to be my next question, of what determines if the constant exist and

**Comment by frett** [comment]
importing translations from OneSky is a semi-manual process at the moment.

OneSky allows us to download all translations as a zip of the strings.xml files. we then just need to move them into the right locations, which i do via `rsync -a`. I then use `git add -p` to spot check the translation updates as they get added to make sure they don't break anything.

**Pull request merged**